text/3-entrees/off-chain-workers/storage.md

```diff
@@ -0,0 +1,102 @@
+# Local Storage in Off-chain Workers
+
+*[`pallets/offchain-demo`](https://github.com/substrate-developer-hub/recipes/tree/master/pallets/offchain-demo)*
+
+Remember we mentioned that off-chain workers cannot write directly to the on-chain storage, that is why they have to submit transactions back on-chain to write to online storage.
+
+Fortunately, there is also a local storage that persist across runs in off-chain workers. It has a similar API usage as [`StorageValue`](/2-appetizers/2-storage-values.html) with `get`, `set`, and `mutate`.
+
+Storage of off-chain workers is persisted across runs of off-chain workers and across nodes with off-chain wokers enabled.
```

Comment:
```suggestion
Storage of off-chain workers is persisted across runs of off-chain workers and blockchain re-organizations.
```
The storage is not shared between the nodes. It's `Local` as the name suggests. I'd also add an example what does it mean to persist across runs and re-orgs.